vm.h

```diff
@@ -46,4 +48,22 @@ void vm_hook_opcode_handler(vm_env *, int, vm_handler);
 
 void vm_run(vm_env *env);
 
+vm_seg_info *vm_new_seg_info(char *mem, size_t sz);
```

Comment:
Is it vital to expose these functions in public header(s)? I meant, I thought they are more ELF-specific than generic VM-oriented.